substrate/frame/contracts/src/schedule.rs

```diff
@@ -396,10 +383,13 @@ impl Default for Limits {
 }
 
 impl<T: Config> Default for InstructionWeights<T> {
-	/// We price both `i64.const` and `drop` as `instr_i64const / 2`. The reason
-	/// for that is that we cannot benchmark either of them on its own.
+	/// We execute 4 different instructions to emulate the `i64.add` therefore
+	/// we have to divide the actual computed gas costs by 4 to have a rough
+	/// estimate as to how expensive each single executed instruction is going to be.
 	fn default() -> Self {
-		Self { base: cost_instr!(instr_i64const, 1), _phantom: PhantomData }
+		let instr_cost = cost!(instr_i64add).ref_time() as u32;
+		let base = instr_cost.saturating_sub(instr_cost / 4);
+		Self { base, _phantom: PhantomData }
```

Comment:
This code does not device the cost by `4`. It does multiple it with `0.75`. So either the comment or the code is wrong. (Or I am wrong)